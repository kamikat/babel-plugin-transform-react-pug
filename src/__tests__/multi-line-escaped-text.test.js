import testHelper from './test-helper';

testHelper(__dirname + '/multi-line-escaped-text.input.js');
