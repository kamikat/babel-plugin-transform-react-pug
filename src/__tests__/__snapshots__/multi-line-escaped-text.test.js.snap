// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JavaScript output: transformed source code 1`] = `"module.exports = <div>foo{\\"\\\\n\\"}{['bar', '123\\\\n\\\\'abc\\\\n'.split('\\\\n').join('\\"').split(\\"'\\").join('@')].join('')}</div>;"`;

exports[`html output: generated html 1`] = `
<div>
  foo
  

  bar123"@abc"
</div>
`;

exports[`static html output: static html 1`] = `
"<div>foo
bar123&quot;@abc&quot;</div>"
`;
